staker-rewards/test/requirements.test.js

```diff
@@ -103,38 +138,137 @@ describe("requirements", async () => {
   })
 
   describe("operator's authorizations check", async () => {
-    beforeEach(async () => {
-      await requirements.checkDeauthorizations()
-    })
-
     it("discovers authorizations on interval start", async () => {
       const operator = "0xA000000000000000000000000000000000000001"
 
       const result = await requirements.checkAuthorizations(operator)
 
-      expect(result).deep.equal(
-        new OperatorAuthorizations(operator, true, true, false)
-      )
+      expect(result).deep.equal({
+        factoryAuthorizedAtStart: true,
+        poolAuthorizedAtStart: true,
+        poolDeauthorizedInInterval: false,
+      })
     })
 
     it("finds if deauthorized during interval", async () => {
```

Comment:
Something I probably missed during the previous review: can we rename this test to `finds if the pool was unauthorized during interval`? The only thing that could get deauthorized is pool, thus the only value that could change is `poolDeauthorizedInInterval`.